src/theme/components/accordion.ts

```diff
@@ -13,6 +13,11 @@ const containerStyles = {
 };
 const buttonStyles = {
   borderWidth: "1px",
+  " > span ": {
+    display: "flex",
+    alignItems: "center",
+    minHeight: "24px",
```

Comment:
If you update this to "20px", you can remove the "6.5px 16px" value in `Accordion.tsx`. I think updating it here and once is better.

Reply:
I'm not sure there's a number for the span height that allows design token spacing for both the case where there's only one accordion button and the case where they're stacked– one of them will need custom padding